Recast/Include/Recast.h

```diff
@@ -277,7 +277,7 @@ struct rcSpan
 {
 	unsigned int smin : RC_SPAN_HEIGHT_BITS; ///< The lower limit of the span. [Limit: < #smax]
 	unsigned int smax : RC_SPAN_HEIGHT_BITS; ///< The upper limit of the span. [Limit: <= #RC_SPAN_MAX_HEIGHT]
-	unsigned int area : 6;                   ///< The area id assigned to the span.
+	unsigned int area : 8;                   ///< The area id assigned to the span.
```

Comment:
There are a few more places that have some implicit dependence on this value, eg. `dtPoly::areaAndtype`. There might also be some other dependencies in the meshing process, though I do not have time to check closer at the moment. But I think you should leave this as is. See also #274

Reply:
Reverted.